src/VirtoCommerce.Platform.Web/Middleware/Hangfire/HangfireUserContextMiddleware.cs

```diff
@@ -0,0 +1,53 @@
+using System.Threading;
+using Hangfire.Client;
+using Hangfire.Server;
+using Microsoft.AspNetCore.Http;
+using VirtoCommerce.Platform.Security;
+
+namespace VirtoCommerce.Platform.Web.Middleware.Hangfire
+{
+    public class HangfireUserContextMiddleware : IClientFilter, IServerFilter
```

Comment:
Add please a one-sentence description of what is main the responsibility of this class.